packages/calcite-components/src/utils/dynamicCSSConst.ts

```diff
@@ -0,0 +1,7 @@
+export const getWidth = (width: string, widthScale: string): string => {
+  return width ? `width-${width}` : widthScale ? `width-${widthScale}` : "";
+};
+
+export const getHeight = (height: string, heightScale: string): string => {
```

Comment:
Both of these helpers have some code in common, so we could add something like the following for both to share or use it directly instead of having 2 similar functions.

```tsx
function getDimension(type: "width" | "height", size: string, scale: string): string {
  return size ? `${type}-${size}` : scale ? `${type}-${scale}` : "";
};
```